apidoc/Titanium/UI/UI.yml

```diff
@@ -1243,6 +1243,30 @@ properties:
     permission: read-only
     platforms: [iphone, ipad]
   
+  - name: TEXT_ELLIPSIZE_TRUNCATE_WORD_WRAP
+    summary: Add ellipses at word boundaries, unless the word itself doesnt fit on a single line
```

Comment:
Use the correct sign there as we do in related docs. Search `UI.yml` for other usages and copy it from there.